### ISSUE TITLE
Feature Request: Allow ignoring trailing commas in single-line array literals

### ISSUE THREAD

**Issue by akicho8** [rubocop/rubocop#14097]
## Is your feature request related to a problem? Please describe.

Given the following code:

```ruby
A = [
  1,
  2
]

B = [1,2,]
```

When the following rule is applied for auto-correction:

```
Style/TrailingCommaInArrayLiteral:
  EnforcedStyleForMultiline: comma
```

The result is:

```ruby
A = [
  1,
  2,
]

B = [1,2]
```

I appreciate the change for A. However, for B, I intentionally added the trailing comma for efficiency purposes, and I would prefer that B be left unchanged.

## Describe the solution you'd like

Since I'm not very familiar with Rubocop, I initially consulted ChatGPT, which suggested the following configuration:

```
Style/TrailingCommaInArrayLiteral:
  EnforcedStyleForMultiline: comma
  EnforcedStyleForSingleline: ignore
```

However, when I tried this, I received an error because the EnforcedStyleForSingleline setting doesn't exist. Despite this, I believe such a feature would be ideal.

The same issue applies to Style/TrailingCommaInHashLiteral and Style/TrailingCommaInArguments as well.

## Describe alternatives you've considered

The alternative I considered is the configuration mentioned above.

## Additional context

```
$ rubocop --version
1.75.2

$ ruby -v
ruby 3.4.2 (2025-02-15 revision d2930f8e7a) +PRISM [arm64-darwin24]
```
